tests/functional/processes/ingest/test_doab_process.py

```diff
@@ -1,8 +1,9 @@
+import pytest
+import os
 from processes import DOABProcess
 from .assert_ingested_records import assert_ingested_records
 from .assert_uploaded_manifests import assert_uploaded_manifests
 
-
```

Comment:
I recommend removing the whitespace changes - it's common practice to leave two new lines before each def.